themes/ddbasic/template.php

```diff
@@ -1299,3 +1246,50 @@ function ddbasic_libraries_info() {
     ),
   );
 }
+
+/**
+ * Implements hook_metatag_metatags_view_alter().
+ */
+function ddbasic_metatag_metatags_view_alter(&$output, $instance, $options) {
```

Comment:
Please revisit the codestyle of this function. There are many violations e.g. spacing after `if`, missing newline etc.